accelerator/core/serializer/serializer.h

```diff
@@ -236,17 +236,29 @@ status_t recv_mam_message_req_deserialize(const char* const obj, ta_recv_mam_req
 /**
  * @brief Serialize response of mam message
  *
- * @param[in] payload_array Response of payload message in utarray
- * @param[out] obj message array formed in JSON
+ * @param payload_array[in]  Response of payload message in 'ta_recv_mam_res_t' datatype
```

Comment:
Should it be `param[in] xxxx` or `param xxxx[in]`?

Reply:
The latter one is better